src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size = 0;
```

Comment:
variable size will be initialized with default value of 0 during object's creation, so you don't need to assign it with 0 here.
```suggestion
    private int size;
```